ci/ci/ci.py

```diff
@@ -152,7 +152,10 @@ async def prod_deploy(request, userdata):
     watched_branch = WatchedBranch(
         0, FQBranch.from_short_str('populationgenomics/hail:main'), True
     )
-    watched_branch.sha = 'HEAD'
+    if params.get('sha'):
```

Comment:
Let's just fail here if the parameter isn't set.

Reply:
But probably we shouldn't allow `HEAD`.

Reply:
I thought it might still be useful to trigger prod_deploy manually. So defaulting to HEAD would be helpful for such scenarios.

Reply:
Given how much headache `HEAD` has caused, I'd vote for making `sha` required. But I'll leave it up to you :).

Reply:
Alright, let's require `sha` :)